tests/playbooks/tests_team_exists.yml

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  tasks:
+    - name: remove the NetworkManager-team package
+      package:
+        name: "NetworkManager-team"
+        state: absent
+
+    - name: "get the rpm package facts"
+      package_facts:
+        manager: "auto"
+
+    - name: "NetworkManager-team checking before team configuration"
+      debug:
+        msg: "NetworkManager-team is not installed before team configuration"
+      when: "'NetworkManager-team' not in ansible_facts.packages"
+
+    - name: "Team interface configuration"
+      include_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          # Specify the team profile
+          - name: team0
+            state: up
+            type: team
+            interface_name: team0
+            # ip configuration (optional)
+            ip:
+              address:
+                - "192.0.2.24/24"
+                - "2001:db8::23/64"
+
+    - name: "get the rpm package facts"
+      package_facts:
+        manager: "auto"
+
+    - name: "NetworkManager-team checking after team configuration"
+      debug:
+        msg: "NetworkManager-team is installed after team configuration"
+      when: "'NetworkManager-team' in ansible_facts.packages"
```

Comment:
This should be an assert not a debug message. If the package is not installed it should fail which is not happening now.

Reply:
updated